netdisco/discoverables/sercomm.py

```diff
@@ -0,0 +1,15 @@
+"""
+Discover Sercomm network cameras.
+These are rebranded as iControl and many others, and are usually
+distributed as part of an ADT or Comcast/Xfinity monitoring package.
+https://github.com/edent/Sercomm-API
+"""
+from . import SSDPDiscoverable
+
+
+class Discoverable(SSDPDiscoverable):
+    """Add support for discovering camera services."""
+
+    def get_entries(self):
+        """Get all the Wireless Network Camera uPnP entries."""
+        return self.find_by_st("urn:schemas-upnp-org:device:Wireless Network Camera:1")
```

Comment:
This is too generic. This does not only apply to sercomm.

Reply:
Unfortunately this is the generic schema they've adopted. They don't appear to have anything more specific I could filter on. How would you recommend handling this?

Reply:
Check out the device.xml file to see if its contents could reveal some more specific parts.

Reply:
Alright, switched to use manufacturer, although I suspect some rebranded devices might use a different string. I can put in a PR if anyone reports additional values.